assets/src/stories-editor/components/block-mover/draggable.js

```diff
@@ -161,10 +161,7 @@ class Draggable extends Component {
 		}
 
 		if ( newSnapLines.length ) {
-			const hasSnapLine = ( item ) => snapLines.find( ( snapLine ) => isShallowEqual( item[ 0 ], snapLine[ 0 ] ) && isShallowEqual( item[ 1 ], snapLine[ 1 ] ) );
-			if ( ! newSnapLines.every( hasSnapLine ) ) {
-				setSnapLines( ...newSnapLines );
-			}
+			setSnapLines( newSnapLines );
```

Comment:
All these minor optimizations are completely irrelevant at this point. Removed them all to just keep the most obvious approach.

Reply:
Makes sense!